packages/acceptit-Tests.package/ACLibraryParserTest.class/instance/testItParsesAStep.st

```diff
@@ -0,0 +1,6 @@
+as yet unclassified
+testItParsesAStep
```

Comment:
So the bug was caused by the parser so you added a test for that particular failing parser method?
Seems fine to me because it's really hard to regression test that debugging a step works, I guess.